src/get-started/extending-and-modifying-components/index.md.njk

```diff
@@ -0,0 +1,146 @@
+---
+layout: layout-pane.njk
+title: Extending and modifying components in production
+section: Get started
+theme: How to guides
+order: 5
+description: What to do when you want to extend or modify a component in production.
+---
+
+You can extend or modify components in the GOV.UK Design System. You could do this to improve a component based on user research or because you have a specific use case for a component in your service.
+
+If you have a production service consider whether your changes:
+- help the long term maintenance of your service
+- allow you to safely install updates from the GOV.UK Design System
+- reduce the risk of technical debt
+
+When you extend or modify components in the GOV.UK Design System you create potential risk. For example, your code or service may break when you then install Design System updates.
+
+You can help reduce potential risk to your code by:
+
+- using [override classes](#start-with-override-classes)
+- not [overwriting GOV.UK Design System code](#avoid-overwriting-govuk-design-system-code)
+- using a [unique prefix for component names][prefix]
+- creating [custom override classes for multiple components](#custom-override-classes)
+- using [BEM for small modifications to components](#small-modifications-to-components)
+- forking [large modifications to components](#large-modifications-to-components)
+
+We use many of these techniques to make sure our code in the GOV.UK Design System does not unintentionally break your application code.
```

Comment:
Avoid using "we" in design system content. It waters down the message that this is community owned.

```suggestion
The GOV.UK Design System team uses many of these techniques to make sure code in the GOV.UK Design System does not unintentionally break your application code.
```